app/diagnostics.go

```diff
@@ -593,7 +602,7 @@ func (a *App) trackConfig() {
 		"trace":                             *cfg.ElasticsearchSettings.Trace,
 	})
 
-	a.SendDiagnostic(TRACK_CONFIG_PLUGIN, map[string]interface{}{
+	pluginConfigData := map[string]interface{}{
 		"enable_antivirus":              pluginActivated(cfg.PluginSettings.PluginStates, "antivirus"),
 		"enable_autolink":               pluginActivated(cfg.PluginSettings.PluginStates, "mattermost-autolink"),
 		"enable_aws_sns":                pluginActivated(cfg.PluginSettings.PluginStates, "com.mattermost.aws-sns"),
```

Comment:
`com.mattermost.aws-sns` is not a prepackaged plugin. @aaronrothschild Should it be prepackaged? Or should be just remove the plugin from this list?

Reply:
For context: It was added in https://github.com/mattermost/mattermost-server/pull/11026

Reply:
I believe it is still not prepackaged but the data is being collected as the other prepackaged plugins.